fix capitalization (#34)
AdobeDocs/create-embed-sdk: src/pages/faq/faq.md

<Accordion>

<AccordionItem header="What is the full editor? " isChevronIcon >

> The full editor component allows developers to embed Adobe Express as an iframe in their own applications. Users can work within this modal editor as they would in Adobe Express, and their designs can be saved back to the integrating application as images.

Reviewer: Let's maybe be a bit more direct? What do you think:

```suggestion
The full editor component allows you to embed Adobe Express as an iframe in their own applications. Users can work within this modal editor as they would in Adobe Express, and their designs can be saved back to the integrating application as images.
```

</AccordionItem>

<AccordionItem header="What are quick actions?" isChevronIcon >

> Adobe-powered quick actions take the most complex tasks from our flagship apps -- like remove background -- and turns them into one-click tasks. The quick actions component is embedded onto your application as an iframe. Once the desired action is complete, the SDK provides two native export buttons to surface to users: 1) download the modified asset or 2) edit further in the full editor component. Developers can also configure their own custom export button to perform tasks such as saving the asset back onto the application.

Reviewer: ```suggestion
Adobe-powered quick actions take the most complex tasks from our flagship apps -- like remove background -- and turns them into one-click tasks. The quick actions component is embedded onto your application as an iframe. Once the desired action is complete, the SDK provides two native export buttons to surface to users: 1) download the modified asset or 2) edit further in the full editor component. You can also configure your own custom export button to perform tasks such as saving the asset back into the application.
```

</AccordionItem>

<AccordionItem header="Do you have a mobile SDK?" isChevronIcon >

> Our efforts are currently focused on desktop and web.

Reviewer: ```suggestion
Our efforts are currently focused on desktop apps and web.
```

Not sure if "desktop" can be used as a standalone word 🤔

</AccordionItem>

<AccordionItem header="Do you support video editing?" isChevronIcon >

> We currently have video quick actions available. As Adobe Express incorporates additional video editing capabilities, our partners and partner end-users will benefit from new features available in Adobe Express.

Reviewer: ```suggestion
We currently have video quick actions available. As Adobe Express incorporates additional video editing capabilities, you and your end-users will benefit from new features available in Adobe Express.
```

<Accordion>

<AccordionItem header="Do I need to embed both the full editor and quick actions?" isChevronIcon >

> No, the full editor and quick actions are separate components and developers can choose to only embed the editor (or vice versa) in their applications.

Reviewer: ```suggestion
No, the full editor and quick actions are separate components and you can choose to only embed the editor (or vice versa) in your applications.
```

</AccordionItem>

<AccordionItem header="How do I get started? Do I need to apply?" isChevronIcon >

If you’re a developer ready to get started, read the [quickstart](https://developer.adobe.com/embed-sdk/docs/guides/) guide in our developer documentation. If you have any questions or would like to get in touch with our partnerships team, please fill out [this interest form](https://survey.adobe.com/jfe/form/SV_9LggYGfzm9w4Yaq) and a member of our team will get in contact.  

</AccordionItem>

<AccordionItem header="How does the full editor component provided by the SDK differ from the free/paid versions of [Adobe Express](https://adobe.com/express)? " isChevronIcon >

There are two functional differences in the embedded Adobe Express full editor compared to Adobe Express on the web:

* Content sharing: In an embedded experience, we assume that a user will save the outcome of their creative project back to the host application in order to continue work in the host application. We have disabled sharing capabilities in embedded Adobe Express experiences.  

> * Content publishing: For the same reason above, a developer has the ability to pre-define the output type (file types, pixel size, etc.) of the Adobe Express project that is saved to the host application. For this reason, post scheduling and exporting multiple output types are disabled for the embedded Adobe Express editor.  

Reviewer: ```suggestion
* Content publishing: For the same reason above, you have the ability to pre-define the output type (file types, pixel size, etc.) of the Adobe Express project that is saved to the host application. For this reason, post scheduling and exporting multiple output types are disabled for the embedded Adobe Express editor.  
```

</AccordionItem>

<AccordionItem header="My user base includes minors/students. How can I be assured they get appropriate content?" isChevronIcon >

> The embedded Adobe Express full editor can be setup to launch student safe content even before a student is signed into the embedded experience. More information about Adobe & Student Privacy can be found here: https://www.adobe.com/privacy/student-policy.html  

Reviewer: ```suggestion
The embedded Adobe Express full editor can be setup to launch student safe content even before a student is signed into the embedded experience. [More information about Adobe & Student Privacy](https://www.adobe.com/privacy/student-policy.html)  
```
